test/wrappers/ortools_test.rb

```diff
@@ -5770,4 +5770,27 @@ def test_self_selection_should_not_change_vehicle_ids
       end
     end
   end
+
+  def test_quantity_precision
+    problem = VRP.lat_lon_two_vehicles
+    problem[:services].each{ |service|
+      service[:quantities] = [{ unit_id: 'kg', value: 1.001 }]
+    }
+    problem[:vehicles].each{ |vehicle|
+      vehicle[:capacities] = [{ unit_id: 'kg', limit: 13 }]
+    }
+
+    result = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:ortools] }}, TestHelper.create(problem), nil)
+    assert_equal 2, result[:routes].size
```

Comment:
Actually we can remove this check, what if the solver returns 1 route with some services unassigned... the test will give false negatives because of this check.

All we care in this tests is the check you already did with the number of activities and maybe we can add a check from the `problem` point of view to be sure that we didn't mess the quantity values internally on the solution we return and the following is still correct (but your check even covers this case, I think) 

`route[:activities].select{ |a| a[:service_id] }.sum{ |a| problem[:services].find{ |s| s[:id] == a[:id]}[:quantities].first[:value] } < 13`